carmajava/launch/saxton_cav_src.launch

```diff
@@ -325,8 +325,11 @@
                to="/$(arg TOP_NS)/$(arg GUIDE_NS)/outgoing_mobility_response"/>
         <remap from="outgoing_mobility_operation"
                to="/$(arg TOP_NS)/$(arg GUIDE_NS)/outgoing_mobility_operation"/>
-        <remap from="get_drivers_with_capabilities"
-               to="/$(arg TOP_NS)/$(arg INTR_NS)/get_drivers_with_capabilities"/>
+        <remap from="inbound_binary_msg"
+               to="/$(arg TOP_NS)/$(arg DRIVER_NS)/dsrc/comms/inbound_binary_msg"/>
+        <remap from="outbound_binary_msg"
+               to="/$(arg TOP_NS)/$(arg DRIVER_NS)/dsrc/comms/outbound_binary_msg"/>
```

Comment:
I don't think we should remap to private driver name spaces. It would be better to update every driver to publish to public topics. Similar to how it will be in CARMA3

Reply:
Same for all below